bin/population/pycbc_population_rates

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2017 Vaibhav Tiwari
+
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Given an xml(.gz) table with CBC injections, this script separates them into:
+(1) (potentially) found injections
+(2) injections that we expect to miss
+The two sets are stored into two separate output files. 
+"""
+
+__author__ = "Vaibhav Tiwari"
+__email__ = "vaibhav.tiwari@ligo.org"
+__version__ = "0.0"
+__date__ = "31.10.2017"
+
+import logging
+import argparse
+import pycbc.version
+
+import os
+import h5py
+import pystan
+import numpy as np
+import scipy.stats as ss
+from pycbc.population import scale_injections as si
+from pycbc.population import rates_functions as rf
+import matplotlib.pyplot as plt
+
+import seaborn as sns
+import matplotlib as mpl
+
+# Parse command line
+parser = argparse.ArgumentParser(description='Estimate rates for flat in log and power-law mass distribution.')
```

Comment:
@vaibhavtewari a minor fix here to avoid future confusion: could you improve the docstring at the top of the file (the current one seems out of date or incomplete) and then replace this line with:
```
parser = argparse.ArgumentParser(description=__doc__)
```

Reply:
@titodalcanton this is unclear to me. The current docstring for description is 
parser = argparse.ArgumentParser(description = "Estimate rates for flat in "
                                      "log and power-law mass distribution.")

Are you talking about something else ?